cli/upload/upload.go

```diff
@@ -76,10 +77,27 @@ func runUploadCommand(command *cobra.Command, args []string) {
 	if len(args) > 0 {
 		path = args[0]
 	}
-
 	sketchPath := arguments.InitSketchPath(path)
-	sk := arguments.NewSketch(sketchPath)
-	discoveryPort := port.GetDiscoveryPort(instance, sk)
+
+	// .pde files are still supported but deprecated, this warning urges the user to rename them
+	if files := sketch.CheckForPdeFiles(sketchPath); len(files) > 0 && importDir == "" && importFile == "" {
+		feedback.Error(tr("Sketches with .pde extension are deprecated, please rename the following files to .ino:"))
+		for _, f := range files {
+			feedback.Error(f)
+		}
+	}
```

Comment:
Why not use `github.com/arduino/arduino-cli/cli/arguments.WarnDeprecatedFiles` here to avoid the code duplication?